api/src/main/resources/metadata/Concepts_MCH-1.xml

```diff
@@ -1,3 +1,323 @@
+  <concept concept_id="129218" retired="false" datatype_id="4" class_id="4" is_set="false" creator="1" date_created="2007-11-03 00:00:00.0" version="" changed_by="1" retired_by="1" uuid="129218AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="135361" retired="false" datatype_id="4" class_id="4" is_set="false" creator="1" date_created="2007-11-03 00:00:00.0" version="" changed_by="1" date_changed="2010-12-14 03:29:10.0" retired_by="1" uuid="135361AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="148834" retired="false" datatype_id="4" class_id="4" is_set="false" creator="1" date_created="2007-11-03 00:00:00.0" version="" changed_by="1" date_changed="2016-07-10 03:57:58.0" retired_by="1" uuid="148834AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="159376" retired="false" datatype_id="6" class_id="7" is_set="false" creator="1" date_created="2010-08-24 14:31:09.0" version="" changed_by="1" date_changed="2016-07-20 21:44:47.0" retired_by="1" uuid="159376AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="161033" retired="false" datatype_id="2" class_id="5" is_set="false" creator="1" date_created="2012-08-03 13:32:47.0" version="" changed_by="1" date_changed="2016-07-10 02:21:35.0" retired_by="1" uuid="161033AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="1067" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2005-01-06 00:00:00.0" version="" changed_by="1" date_changed="2016-08-16 07:46:41.0" retired_by="1" uuid="1067AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="125872" retired="false" datatype_id="2" class_id="4" is_set="false" creator="1" date_created="2007-11-03 00:00:00.0" version="" changed_by="1" date_changed="2016-07-10 02:23:05.0" retired_by="1" uuid="125872AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="126127" retired="false" datatype_id="4" class_id="4" is_set="false" creator="1" date_created="2007-11-03 00:00:00.0" version="" changed_by="1" date_changed="2016-07-10 02:55:15.0" retired_by="1" uuid="126127AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="151849" retired="false" datatype_id="4" class_id="4" is_set="false" creator="1" date_created="2007-10-14 07:41:45.0" version="" changed_by="1" date_changed="2016-06-19 20:10:01.0" retired_by="1" uuid="151849AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="159896" retired="false" datatype_id="4" class_id="2" is_set="false" creator="1" date_created="2010-11-24 18:11:43.0" version="" changed_by="1" date_changed="2010-11-24 18:17:31.0" retired_by="1" uuid="159896AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="1065" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2005-01-06 00:00:00.0" version="" changed_by="1" date_changed="2016-08-16 07:47:47.0" retired_by="1" uuid="1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
```

Comment:
In my latest commit, I have stripped off from the XML, all concept metadata definitions i.e data types,concept classes etc and removed concept definitions that already exist on UgandaEMR like 1065,1066,1067,126127